### DISCUSSION TITLE
dismissing alarm from garmin watch does not stop alarm on phone

### DISCUSSION THREAD

**Discussion by jhnc** [FossifyOrg/General-Discussion#642] [Q&A]
I have a Samsung S7 connected to a Pebble watch.
I have a Pixel 8 running GrapheneOS connected to a Garmin Forerunner 55.
On each phone, I have the system-provided clock app; the Calengoo calendar app; and Fossify Clock.

On both systems, if I set an alarm using the system-provided clock app, nothing appears on the watch.

On both systems, if I set an alarm using Calengoo, the watch is notified, and dismissing on the watch stops the alarm on the phone.

With S7/Pebble, if I set an alarm using Fossify Clock, the watch is notified, and dismissing on the watch stops the alarm on the phone.

With GrapheneOS/Garmin, if I set an alarm using Fossify Clock, the watch is notified, but dismissing on the watch does not stop the alarm on the phone.

Is this a shortcoming of the Garmin notification system, or a bug in Fossify Clock?

**Comment by Aga-C** [comment]
Please raise a bug report for it.
